developing/network/join-mainnet.md

```diff
@@ -20,7 +20,37 @@ Download and place the genesis file in the osmosis config folder:
 wget -O ~/.osmosisd/config/genesis.json https://github.com/osmosis-labs/networks/raw/main/osmosis-1/genesis.json
 ```
 
-## Set Up Cosmovisor
+## Latest Version (V10) Upgrade Info
```

Comment:
This section can be left out, it was more so just for validators in the moment, it wont matter for people syncing at this point in time

Reply:
Agreed, all feedback applied